src/status_im2/common/scroll_page/view.cljs

```diff
@@ -41,65 +28,98 @@
    (max minimum)
    (min maximum)))
 
-(defn icon-top-fn
-  [scroll-height]
-  (if (<= scroll-height negative-scroll-position-0)
-    -40
-    (->> (+ scroll-position-0 scroll-height)
-         (* (if platform/ios? 3 1))
-         (+ -40)
-         (min 8))))
-
-(defn icon-size-fn
-  [scroll-height]
-  (->> (+ scroll-position-0 scroll-height)
-       (* (if platform/ios? 3 1))
-       (- max-image-size)
-       (max min-image-size)
-       (min max-image-size)))
+(defn scroll-page-header
+  [scroll-height name page-nav cover sticky-header]
+  [:f>
+   (fn []
+     (let [input-range         (if platform/ios? [-47 10] [0 150])
+           output-range        (if platform/ios? [-100 0] [-169 -45])
+           y                   (animated/value @scroll-height)
+           translate-animation (animated/interpolate y
+                                                     {:inputRange  input-range
```

Comment:
FYI: See the implementation of `animated/interpolate`. You don't need camel case in keywords.

Reply:
We should use `reanimated/interpolate` and `reanimated/use-shared-value` if possible btw